lib/signaling/v2/room.js

```diff
@@ -752,9 +752,8 @@ function replaceNullsWithDefaults(activeIceCandidatePair, peerConnectionId) {
     port: 0,
     priority: 0,
     protocol: 'udp',
-    relayProtocol: 'udp',
     url: ''
-  }, filterObject(activeIceCandidatePair.localCandidate || {}, null));
+  }, filterObject(activeIceCandidatePair.localCandidate || {}, null), (activeIceCandidatePair.localCandidate && activeIceCandidatePair.localCandidate.relayProtocol && { relayProtocol: 'udp' }));
```

Comment:
Ah, I see. 

The change was to ensure that regardless of what we give to `relayProtocol` it will be `udp`. In my tests, I give 
`{ relayProtocol: 'bar' }`. But I'll remove it! :)